packages/core/custom-elements-lsp/src/parser/analyze.ts

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
```

Comment:
This is just the old `analyze.js` script which has been turned into a typescript file and moved